PyApps/src/Plugins/display_image_qt5.py

```diff
@@ -24,76 +24,85 @@
-#  You should have received a copy	 Vous devez avoir recu une copie
-#  of the GNU General Public		 de la Licence Publique Generale
-#  License along with this		 GNU en meme temps que ce
-#  software; if not, contact the	 logiciel ; si ce n'est pas le
-#  Free Software Foundation, Inc.	 cas, communiquez avec la Free
-#  at http://www.fsf.org.		 Software Foundation, Inc. au
-#						 http://www.fsf.org.
+#  You should have received a copy         Vous devez avoir recu une copie
+#  of the GNU General Public                 de la Licence Publique Generale
+#  License along with this                 GNU en meme temps que ce
+#  software; if not, contact the         logiciel ; si ce n'est pas le
+#  Free Software Foundation, Inc.         cas, communiquez avec la Free
+#  at http://www.fsf.org.                 Software Foundation, Inc. au
+#                                                 http://www.fsf.org.
 #
-#  email:				 courriel:
-#  business@hia-iha.nrc-cnrc.gc.ca	 business@hia-iha.nrc-cnrc.gc.ca
+#  email:                                 courriel:
+#  business@hia-iha.nrc-cnrc.gc.ca         business@hia-iha.nrc-cnrc.gc.ca
 #
-#  National Research Council		 Conseil national de recherches
-#      Canada				    Canada
-#  Herzberg Institute of Astrophysics	 Institut Herzberg d'astrophysique
-#  5071 West Saanich Rd.		 5071 West Saanich Rd.
-#  Victoria BC V9E 2E7			 Victoria BC V9E 2E7
-#  CANADA					 CANADA
+#  National Research Council                 Conseil national de recherches
+#      Canada                                    Canada
+#  Herzberg Institute of Astrophysics         Institut Herzberg d'astrophysique
+#  5071 West Saanich Rd.                 5071 West Saanich Rd.
+#  Victoria BC V9E 2E7                         Victoria BC V9E 2E7
+#  CANADA                                         CANADA
 #
 
 import sys
 
-from PyQt4 import Qt
-import PyQt4.Qwt5 as Qwt
+from qwt.qt.QtGui import (QApplication, QDialog, QGridLayout,QHBoxLayout,QToolTip,QPrinter,QPrintDialog,
+         QLabel, QSizePolicy, QSlider, QPushButton, QVBoxLayout, QSpinBox, QSpacerItem)
+from qwt.qt.QtGui import QPen, QColor,QWidget, QImage, qRgba, QFont, QFontInfo, QMenu, QActionGroup, QAction, QMessageBox
+from qwt.qt.QtCore import Qt, QSize, QObject, pyqtSignal, QTimer
+from qwt import (QwtPlot, QwtPlotMarker, QwtLegend, QwtPlotGrid, QwtPlotCurve,QwtPlotRenderer,
+                 QwtPlotItem, QwtText, QwtLegendData, QwtLinearColorMap, QwtSymbol,
+                 QwtInterval, QwtScaleMap, QwtScaleDraw, QwtScaleDiv, toQImage)
+from qwt import  QwtLogScaleEngine, QwtLinearScaleEngine
 
-from QwtSpy_qt4 import *
+
+
+from QwtSpy_qt5 import *
 
 import numpy
 import math
 
-#from UVPAxis import *
-#from ComplexColorMap import *
-from ComplexScaleDraw_qt4 import *
-from QwtPlotCurveSizes_qt4 import *
-from QwtPlotImage_qt4 import *
-from VellsTree_qt4 import *
-from Timba.GUI.pixmaps import pixmaps
-#from guiplot2dnodesettings import *
+from ComplexScaleDraw_qt5 import *
```

Comment:
Instead of * imports which pulls in other imported namespaces import only the necessary functions

Reply:
done

Reply:
Now modified to include QwtPlotCurveSizes_qt5 which should allow plotting of various Solver fitting parameters - not that I have much stuff for testing :(